src/main/webapp/PlaceGuideOnList.js

```diff
@@ -0,0 +1,41 @@
+class PlaceGuideOnList {
+  constructor(placeGuideId, place, name, creator, description, audioKey, audioLength, imageKey, createdByCurrentUser) {
+    const placeGuideDiv = document.createElement("div");
+    placeGuideDiv.setAttribute('id', placeGuideId);
```

Comment:
Consider using an id which is specific for PlaceGuideOnLists only. For example "placeGuideOnList-{placeGuideId}"

Reply:
changed to `placeGuideOnList-{placeGuideId}`